modules/migration-installing-migration-operator-manually.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id="installing-migration-operator-manually_{context}"]
+= Installing the CAM Operator manually
+
+You can install the CAM Operator manually on an {product-title} 3 cluster.
+
+[NOTE]
+====
+You can install the CAM Operator manually on an {product-title} 4 cluster, but it is normally installed with OLM.
+====
+
+.Procedure
+
+. Download the *_operator.yml_* file:
```

Comment:
File names should be in backticks. (See other instances in this file and throughout PR)

Reply:
Duplicate comment. I've left the other one open.